scripts/check.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
+# Run flake8 against the Django codebase
+
 set -e
 set -x
 
-# Run flake8 against the Django codebase and output a known string so that
```

Comment:
I don't want to loose the comment explaining why ``|| echo flake8`` is there. We could breakup the comment and move the details below the ``set -x`` to clean up the help output.